### PR TITLE
feat: delete credential

### LINKED ISSUE #227: Delete credential
### Description

A user should have the possibility to fully remove a credential from their wallet.

While this was not important in the early phases of development and for demo environments, it is now becoming more of an inconvenience during testing not to be able to recreate a flow again.

The user should be prompted with a confirmation screen before the credential is actually removed (this could be turned off in _dev mode_). In this confirmation screen, the user could be asked if they want to keep the associated history for that credential or if it should also be removed.

### Motivation

Required for production release, but would already improve DX in current stage.

### Requirements

1. New action & reducer to trigger removal (parameters: `credential_id`, `remove_history: bool`)
2. UI element ("three dot menu") in credential/badge details to trigger confirmation sheet
3. New UI element: "confirmation sheet" (switch/checkbox to confirm removal of history)

### Open Questions

* Recoverability? Undo delete? Restore from trash?
* Batch delete? Delete multiple credentials/badges at once?

### Are you planning to contribute this in a PR?

Yes

### PR DESCRIPTION
**Pull request by daniel-mader**
# Description of change
Credentials can be deleted via the menu at the top right corner.

## Links to any relevant issues
- closes #227

## How the change has been tested
Local development, app builds on physical devices.

## Definition of Done checklist
Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Oran-Dan — DISMISSED** [review]
Nice work!

File: unime/src/i18n/en/index.ts [hunk 1]
```diff
@@ -270,6 +270,14 @@ const en = {
       DESCRIPTION: 'Description',
       CONTENTS: 'Contents',
     },
+    ACTIONS: {
+      DELETE: {
+        BUTTON_LABEL: 'Delete credential',
+        TITLE: 'Delete credential',
+        DESCRIPTION: 'Are you sure you want to remove this credential from your wallet? This action cannot be undone.',
```
Comment (maiertech) [inline unime/src/i18n/en/index.ts]: Replace 'remove' with 'delete' to be consistent.

File: unime/src/i18n/de-DE/index.ts [hunk 2]
```diff
@@ -271,6 +271,15 @@ const de = {
       DESCRIPTION: 'Beschreibung',
       CONTENTS: 'Inhalt',
     },
+    ACTIONS: {
```
Comment (maiertech) [inline unime/src/i18n/de-DE/index.ts:274]: "Entfernen" is not the same as "löschen". In English we use "delete", therefore it should be "löschen".

**Review by maiertech — COMMENTED** [review]
Check comments for translations.

**Review by maiertech — APPROVED** [review]

**Review by Oran-Dan — APPROVED** [review]

**Pull request merged**